### ISSUE TITLE
Only able to log in manually in Chrome after MFA input

### ISSUE THREAD

**Issue by msimoni18** [mintapi/mintapi#364]
Having trouble logging in with MFA code using ```mintapi=1.58```.

I copied the "from python" section from the README.md and input my username/password. When I run the code, it prompts me for 6-digit MFA code. It continuously asks for the MFA code. In order to log in successfully, I have to manually click the login button in Chrome *after* entering the MFA code for the first time, and *before* the second prompt appears. If the second prompt appears before I can click the login button, the code never finishes. I've also tried adding a path to ```session_path```, but have the same problem.

Is this a bug, or am I doing something wrong?  I'm new to ```mintapi```, so it's entirely possible I'm missing something.

**Comment by burkematthew** [comment]
@msimoni18 can you add a snippet of your code that you're running?

**Comment by xinluh** [comment]
There is a new version of the 2fa page with different id's, e.g. https://github.com/xinluh/yamintapi/commit/b4546d64dd5adf22d2b9e54c088d14da0fe127e4#diff-29db0a202391c066de62d16a0d5b5cc129dc4fb8fb75c5f759a65fdd0452ceef

**Comment by pazwicker** [comment]
Happening to me too. Running:

```python
mint = mintapi.Mint(
        "myemail",  # Email used to log in to Mint
        "mypassword",  # Your password used to log in to mint
        mfa_method="soft-token",  # Can be 'sms' (default) or 'email'.
        headless=True,  # Whether the chromedriver should work without opening a
        mfa_input_callback=None,
        mfa_token='my token',
        session_path=None,
        wait_for_sync=False,  # do not wait for accounts to sync
        wait_for_sync_timeout=1,  # number of seconds to wait for sync
    )
```

**Comment by msimoni18** [comment]
Here's my code that I directly copied from the README.md.  I only changed the username, password, and tried ```session_path=None``` and ```session_path=mypath```.  

```
import mintapi
mint = mintapi.Mint(
    'myusername',  # Email used to log in to Mint
    'mypassword',  # Your password used to log in to mint
    # Optional parameters
    mfa_method='sms',  # See MFA Methods section
                       # Can be 'sms' (default), 'email', or 'soft-token'.
                       # if mintapi detects an MFA request, it will trigger the requested method
                       # and prompt on the command line.
    mfa_input_callback=None,  # see MFA Methods section
                              # can be used with any mfa_method
                              # A callback accepting a single argument (the prompt)
                              # which returns the user-inputted 2FA code. By default
                              # the default Python `input` function is used.
    mfa_token=None,   # see MFA Methods section
                      # used with mfa_method='soft-token'
                      # the token that is used to generate the totp
    intuit_account=None, # account name when multiple accounts are registered with this email.
    headless=False,  # Whether the chromedriver should work without opening a
                     # visible window (useful for server-side deployments)
                         # None will use the default account.
    session_path='/Users/username/.minitap/session', # Directory that the Chrome persistent session will be written/read from.
                       # To avoid the 2FA code being asked for multiple times, you can either set
                       # this parameter or log in by hand in Chrome under the same user this runs
                       # as.
    imap_account=None, # account name used to log in to your IMAP server
    imap_password=None, # account password used to log in to your IMAP server
    imap_server=None,  # IMAP server host name
    imap_folder='INBOX',  # IMAP folder that receives MFA email
    wait_for_sync=False,  # do not wait for accounts to sync
    wait_for_sync_timeout=300,  # number of seconds to wait for sync
	use_chromedriver_on_path=False,  # True will use a system provided chromedriver binary that
	                                 # is on the PATH (instead of downloading the latest version)
  )


# Get transactions
df = mint.get_transactions() # as pandas dataframe
print(df)

# Close session and exit cleanly from selenium/chromedriver
mint.close()
```

**Comment by burkematthew** [comment]
Thanks everyone for the information! I'll take a look at this evening.

**Comment by areagu** [comment]
same thing is happening to me, keeps on asking for MFA code even after entering one.. whether headless=True or not

**Comment by jaron-l** [comment]
This will likely get resolved with #367.

**Comment by pazwicker** [comment]
I just submitted a new PR that fixes this issue for me, #370.

**Comment by chino** [comment]
I've gotten around this as well using a similar approach to the existing PRs here.

I would vote for using the `id=iux-mfa-soft-token-verification-code` change along with the `css=button[data-testid=VerifySoftTokenSubmitButton]` approach as that doesn't depend on strict div layouts (which feels both simpler and more resilient to future change).

**Comment by mrooney** [comment]
Should be resolved w/ #367 and #370!
